redisgears_v8_plugin/src/v8_native_functions.rs

```diff
@@ -882,6 +882,95 @@ pub(crate) fn initialize_globals_1_1(
     Ok(())
 }
 
+fn add_register_function_api(
```

Comment:
I think it is OK that it will be on 1.0, we did not release yet any official version. I think we said that we would not even introduce 1.1 if it wasn't for testing right?